js/views/settingsVw.js

```diff
@@ -323,6 +342,10 @@ module.exports = pageVw.extend({
     return this;
   },
 
+  showPeers: function () {
+    this.$el.find('.js-connectedPeers').toggle();
```

Comment:
As an FYI, Backbone view's give us [dollar function](http://backbonejs.org/#View-dollar) which is short hand for `<view-instance>.$el.find('.blah-blah')`. For example:

```
this.$el.find('.js-connectedPeers').toggle();
```

could be written more concisely as:

```
this.$('.js-connectedPeers').toggle();
```

No need to update... just wanted you to be aware.